eval.go

```diff
@@ -202,9 +187,14 @@ func (e *CallExpr) eval(app *App, args []string) {
 			}
```

Comment:
Unrelated, but shouldn't we be `return`ing in the block above*? If we fail to open the file, the code below might very well panic.

*) Github does not let me comment on lines beyond the diff, pfft.